### DISCUSSION TITLE
ImportError: cannot import name 'build_py_2to3' from 'distutils.command.build_py'

### DISCUSSION THREAD

**Discussion by JorgD379** [keras-team/autokeras#1913] [Q&A]
![image](https://github.com/keras-team/autokeras/assets/83813873/180884b5-f1e8-4fb4-a177-eb65fbcb41d0)
I can't install autokeras, I prescribed pip install autokeras, and this error comes out.Help please

**Comment by olasawa** [comment]
I have the same issue, did you manage to solve it?

**Reply by zz4fap** [reply]
I have the same issue.

**Reply by tomyputw** [reply]
I have the same issue.

**Comment by tomyputw** [comment]
this problem is not caused by autokeras but pip, pip install h5py and that is all.
use pip install autokeras --no-deps
